edgelet/iotedge/src/main.rs

```diff
@@ -323,6 +351,96 @@ fn run() -> Result<(), Error> {
             tokio_runtime.block_on(Logs::new(id, options, runtime()?).execute())
         }
         ("version", _) => tokio_runtime.block_on(Version::new().execute()),
+        ("support-bundle", Some(args)) => {
+            let location = args.value_of("location").unwrap_or_default();
```

Comment:
We'd talked about using `expect` like the `check` subcommand does, since the args have a default value so `value_of` won't return `None`. `unwrap_or_default` is not what you want - it's empty string in this case, 0 for the integer parameters, neither of which make sense given the way they're used.